### DISCUSSION TITLE
Password OpenSign

### DISCUSSION THREAD

**Discussion by KINGDB1** [OpenSignLabs/OpenSign#1237] [Q&A]
I have finished installing OpenSign on my server. What is the default password to use to access the platform, please?
![opensing](https://github.com/user-attachments/assets/c7d52a69-fc2a-45d7-9952-ae05c09e4964)

**Comment by thlindustries** [comment]
hey man, you need to:
- clone the repository; 
- create a file called .env.prod; 
- copy everything from the file .env.local_dev and paste it inside the .env.prod file;
- run docker-compose up

after this, access https://localhost:3001/, click on advanced, accept the risks and boom, you'll be able to use it locally

**Reply by thlindustries** [reply]
![Screenshot from 2025-01-23 11-25-33](https://github.com/user-attachments/assets/9e2d0c1b-b6a5-41de-b01c-23f7f196a4b0)
![Screenshot from 2025-01-23 11-25-26](https://github.com/user-attachments/assets/c9a99bef-8082-401e-a66f-449d26bec13b)


And also, add this variable in the .env.prod file: "REACT_APP_ENABLE_SUBSCRIPTION=true"
